externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Conversion/Passes.td

```diff
@@ -0,0 +1,25 @@
+#ifndef TORCH_MLIR_DIALECTS_CONVERSION_PASSES
+#define TORCH_MLIR_DIALECTS_CONVERSION_PASSES
+
+include "mlir/Pass/PassBase.td"
+
+#ifdef TORCH_MLIR_DIALECTS_ENABLE_TCP
+//===----------------------------------------------------------------------===//
+// TcpToTosa
+//===----------------------------------------------------------------------===//
+
+def ConvertTcpToTosa
+      : Pass<"convert-tcp-to-tosa", "func::FuncOp"> {
+  let summary = "Lower TCP to TOSA";
+  let description = [{
+    Pass that converts TCP operations to the equivalent operations in TOSA.
+  }];
+
+  let constructor = "mlir::torch::tcp::createConvertTcpToTosa()";
```

Comment:
The convention seems to be `createConvertTcpToTosaPass`?

Reply:
Done.